app/main.cpp

```diff
@@ -224,6 +224,17 @@ int main(int argc, char *argv[])
                                 QLatin1String("verbose"));
     parser.addOption(verbose);
 
+    QCommandLineOption maxStack(QLatin1String("max-stack"),
+                                QCoreApplication::translate(
+                                "main", "Maximum size of callchain and branchStack."),
+                                QLatin1String("max-stack"), QLatin1String("127"));
+    parser.addOption(maxStack);
+
+    QCommandLineOption branchTraverse(QLatin1String("branch-traverse"),
+                                      QCoreApplication::translate(
+                                      "main", "Short branchStack resolveCallchain traverse. Concerns lbr."));
```

Comment:
separate patch for this second option please

Reply:
also, please explain why this is required in the first place - if someone recorded with `--call-graph lbr` then we want to use it. why would one ever want to disable this?

Also please rephrase the description. I cannot understand it. From the code this more looks like a `--ignore-branch-stack` option? So something like "Ignore LBR branch stack for callchain construction"?

Reply:
To compute event costs by instructions for Disassembly. For the moment there are only branchStack items are traversed.
It is not enough. I added more frames to traverse. But it can have a certain impact to performance for LBR.
I have entered option branch-traverse to turn off traverse by additional frames for LBR.
So it is proper name for option. When it is passed the only branchStack is traversed and events costs are not showed.